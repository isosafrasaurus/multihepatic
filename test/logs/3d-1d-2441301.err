Running "module reset". Resetting modules to system default. The following $MODULEPATH directories have been removed: None
set -u

APPTAINER_BIN="$(command -v apptainer || true)"
[[ -z "$APPTAINER_BIN" ]] && APPTAINER_BIN="$(command -v singularity || true)"
if [[ -z "$APPTAINER_BIN" ]]; then
  echo "[JOB][ERROR] apptainer/singularity not found" >&2
  exit 127
fi

# Job inputs from the wrapper (expanded below via sed)
IMAGE_URI="docker://ghcr.io/isosafrasaurus/tacc-mvapich2.3-python3.12-graphnics:latest"
REPO_URL="https://github.com/isosafrasaurus/3d-1d"
RUN_SCRIPT="test.py"
TASKS=1

# Use repo in current directory if present; otherwise clone into WORK/SCRATCH for this job
if [[ -f "./${RUN_SCRIPT}" ]]; then
  REPO_DIR="$PWD"
else
  REPO_DIR="${WORK:-${SCRATCH:-$PWD}}/3d-1d-${SLURM_JOB_ID}"
  mkdir -p "$REPO_DIR"
  module load git >/dev/null 2>&1 || true
  if ! command -v git >/dev/null 2>&1; then
    echo "[JOB][ERROR] git is required to clone ${REPO_URL} but was not found." >&2
    exit 2
  fi
  echo "[JOB] Cloning ${REPO_URL} into ${REPO_DIR} ..."
  git clone --depth 1 "$REPO_URL" "$REPO_DIR"
fi
Cloning into '/work2/10756/piercezhang9871/stampede3/3d-1d-2441301'...
cd "$REPO_DIR"
echo "[JOB] Repo dir: $PWD"
ls -la 2>/dev/null || true

# Avoid occasional Apptainer runtime dir quirks on shared systems
unset XDG_RUNTIME_DIR || true
export APPTAINER_CACHEDIR="${SCRATCH:-$HOME}/.apptainer/cache"
mkdir -p "$APPTAINER_CACHEDIR"

# Trace and verify container runtime, then launch via srun so Slurm captures output robustly
set -x
"$APPTAINER_BIN" --version
+ /opt/apps/tacc-apptainer/1.4.1/bin/apptainer --version

# For serial jobs, srun -n 1 is fine; for MPI later, weâ€™d switch to ibrun or srun with more tasks.
srun -n "${TASKS}" --export=ALL \
  "$APPTAINER_BIN" exec -B "$PWD:/workspace" "$IMAGE_URI" \
  python3 "/workspace/$RUN_SCRIPT"
+ srun -n 1 --export=ALL /opt/apps/tacc-apptainer/1.4.1/bin/apptainer exec -B /work2/10756/piercezhang9871/stampede3/3d-1d-2441301:/workspace docker://ghcr.io/isosafrasaurus/tacc-mvapich2.3-python3.12-graphnics:latest python3 /workspace/test.py
INFO:    Converting OCI blobs to SIF format
INFO:    Starting build...
INFO:    Fetching OCI image...
INFO:    Extracting OCI image...
INFO:    Inserting Apptainer configuration...
INFO:    Creating SIF file...
INFO:    To see mksquashfs output with progress bar enable verbose logging
INFO:    gocryptfs not found, will not be able to use gocryptfs
/usr/bin/python3: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /opt/apps/xalt/xalt/lib64/libxalt_init.so)
/usr/bin/python3: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /opt/apps/xalt/xalt/lib64/libxalt_init.so)
srun: error: c455-003: task 0: Exited with exit code 1
