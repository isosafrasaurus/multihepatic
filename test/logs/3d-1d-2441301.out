[JOB] Cloning https://github.com/isosafrasaurus/3d-1d into /work2/10756/piercezhang9871/stampede3/3d-1d-2441301 ...
[JOB] Repo dir: /work2/10756/piercezhang9871/stampede3/3d-1d-2441301
total 31
drwx------ 7 piercezhang9871 G-825393 4096 Oct  6 13:06 .
drwx------ 4 piercezhang9871 G-825393 4096 Oct  6 13:06 ..
-rw------- 1 piercezhang9871 G-825393 6665 Oct  6 13:06 3d_1d.ipynb
drwx------ 2 piercezhang9871 G-825393 4096 Oct  6 13:06 fem
drwx------ 8 piercezhang9871 G-825393 4096 Oct  6 13:06 .git
-rw------- 1 piercezhang9871 G-825393  125 Oct  6 13:06 .gitattributes
-rw------- 1 piercezhang9871 G-825393  103 Oct  6 13:06 .gitignore
-rw------- 1 piercezhang9871 G-825393 8334 Oct  6 13:06 README.md
drwx------ 2 piercezhang9871 G-825393 4096 Oct  6 13:06 src
drwx------ 3 piercezhang9871 G-825393 4096 Oct  6 13:06 tacc
-rw------- 1 piercezhang9871 G-825393 2308 Oct  6 13:06 test.py
drwx------ 2 piercezhang9871 G-825393 4096 Oct  6 13:06 tissue
apptainer version 1.4.1-1.el9
